frontend/src/pages/agreements/approve/ApproveAgreement.hooks.js

```diff
@@ -97,23 +113,62 @@ const useApproveAgreement = () => {
         ? groupByServicesComponent(agreement.budget_line_items)
         : [];
     const budgetLinesInReview = agreement?.budget_line_items?.filter((bli) => bli.in_review) || [];
-    /**
-     *  @typedef {import('../../../components/ChangeRequests/ChangeRequestsList/ChangeRequests').ChangeRequest} ChangeRequest
-     *  @type {ChangeRequest[]}
-     */
     const changeRequestsInReview = /** @type {ChangeRequest[]} */ (
         getInReviewChangeRequests(agreement?.budget_line_items)
     );
+
     const changeInCans = getTotalByCans(budgetLinesInReview);
 
     let statusForTitle = "";
 
     if (changeRequestType === CHANGE_REQUEST_SLUG_TYPES.STATUS) {
-        statusForTitle = `- ${renderField(null, "status", status)}`;
+        statusForTitle = `- ${renderField(null, "status", statusChangeTo)}`;
     }
     const changeRequestTitle = toTitleCaseFromSlug(changeRequestType);
     const title = `Approval for ${changeRequestTitle} ${statusForTitle}`;
 
+    let requestorNoters = "";
+    if (changeRequestType !== CHANGE_REQUEST_SLUG_TYPES.BUDGET) {
+        requestorNoters = changeRequestsInReview[0]?.requestor_notes ?? "";
+    }
```

Comment:
@stevtek is it a safe assumption to just grab the submitters_notes from the first in_review BLI that is not of budget change?

Reply:
I think that collecting all distinct notes is better